tests/phpunit/includes/testcase-trait.php

```diff
@@ -61,6 +61,12 @@ static function create_language( $locale, $args = array() ) {
 		$values['term_group'] = 0; // Default term_group.
 
 		$args = array_merge( $values, $args );
+
+		$links_model     = self::$model->get_links_model();
+		$pll_admin = new PLL_Admin( $links_model );
+		$admin_default_term = new PLL_Admin_Default_Term( $pll_admin );
+		add_action( 'pll_add_language', array( $admin_default_term, 'handle_default_category_on_create_language' ) );
```

Comment:
The goal to put the actions in a dedicated function was not to hook it in this method.

Here's what happens with this code:

- Each time we call `PLL_UnitTestCase_Trait::create_language()` in a `wpSetUpBeforeClass()` method, we are going to register one new 'pll_add_language' action per call.
- Before each test, `WP_UnitTestCase_Base::setUp()`will [back up these actions](https://wordpress.org/plugins/unagi/).
- If we call `PLL_Admin_Default_Term::add_hooks()` during the test ([like it is done here](https://github.com/polylang/polylang/blob/306ae6240e8dcc63a3202d04f92c9169394612a0/tests/phpunit/tests/test-default-term.php#L54-L55)), we will add one more 'pll_add_language' action.
- If we call `PLL_UnitTestCase_Trait::create_language()` during the test ([like it is done here](https://github.com/polylang/polylang/blob/306ae6240e8dcc63a3202d04f92c9169394612a0/tests/phpunit/tests/test-default-term.php#L57-L60)), we will add one more 'pll_add_language' action per call.
- After the test, `WP_UnitTestCase_Base::tearDown()` will replace all the hooks registered during the tests by the hooks backed up before the test: https://github.com/WordPress/wordpress-develop/blob/3d8a5fd839d3eb1642a38f25e4d2901b009371e2/tests/phpunit/includes/abstract-testcase.php#L168 (this means that it keeps the 'pll_add_language' action we hooked during `wpSetUpBeforeClass()`)

So we will end up with too duplicated hooks in every scenario. Couldn't you call directly the `handle_default_term_on_create_language()` method with the correct argument directly?